lib/src/icon_toggle.dart

```diff
@@ -25,18 +24,18 @@ class IconToggle extends StatefulWidget {
   final Color activeColor;
   final Color inactiveColor;
   final bool value;
-  final ValueChanged<bool> onChanged;
+  final ValueChanged<bool>? onChanged;
```

Comment:
This should not be nullable because on `_position.addStatusListener()` you are making use of it